<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image :</string>
    </property>
   </widget>
   <widget class="QLabel" name="pbxSrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>176</width>
      <height>220</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabLCD">
     <attribute name="title">
      <string>LCD</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Display Select</string>
      </property>
      <widget class="QRadioButton" name="rdbCS1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCS2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCS3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCS4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCS5">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCS6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCSA">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnPlay">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Play One</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoad">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Load...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClear">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInital">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Inital</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnShow">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnColorBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Bar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlayOmni">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Omni</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIO">
     <attribute name="title">
      <string>I/O</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>I/O Test</string>
      </property>
      <widget class="QCheckBox" name="ckbWR">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>WR</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbRS">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>RS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbDB0">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>DB0</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbIOTest">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbCS1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>CS1</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbRST">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>RST</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ckbBL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>BL</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnIOTest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPano">
     <attribute name="title">
      <string>Pano</string>
     </attribute>
     <widget class="QPushButton" name="btnConvert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlayPano">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Pano</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ckbDir">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Out to In</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnShowPano">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Pano</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="pgbConvert">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
